### PR TITLE
[media] Consistency across platforms splash screens (avoid versions)

### PR DESCRIPTION
**Pull request by enen92**
## Description
This is an alternative to https://github.com/xbmc/xbmc/pull/24026 and aims at bringing consistency across all platforms that require a load splashscreen (android, tvos, ios, webos). Try to use a common image in png format without any reference for the specific Kodi version (that is shown afterwards on the windowing system/kodi is actually loading) to make the release process easier.

## Motivation and context
Save a few hair to @fuzzard.
Webos requires a splashscreen in png format and doing that for every release is a PITA.

## How has this been tested?
Runtime tested on android

## What is the effect on users?
None really, the new image is shown on those platforms while the app is starting. Saves a few bytes on the overall package (at least on android).

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

### CODE REVIEW

File: xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/Contents.json [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
+      "filename" : "splashscreen.png",
       "extent" : "full-screen",
       "minimum-system-version" : "9.0",
       "scale" : "1x"
     },
     {
       "orientation" : "landscape",
       "idiom" : "tv",
-      "filename" : "splash@2x.jpg",
+      "filename" : "splashscreen@2x.png",
```
Comment (da-anda) [inline xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/Contents.json:14]: unrelated to your change, but there was no, and now also is no `@2x` splash image. So shouldn't this entire config block get removed, or is this file autogenerated during packaging? @fuzzard?

File: tools/webOS/packaging/appinfo.json.in [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "title": "@APP_NAME@",
   "icon": "icon.png",
   "largeIcon": "largeIcon.png",
-  "splashBackground": "media/splash_webOS.png",
+  "splashBackground": "splashscreen.png",
```
Comment (da-anda) [inline tools/webOS/packaging/appinfo.json.in:10]: shouldn't this be `media/splashscreen.png`?

**Review by da-anda — COMMENTED** [review]
can't say much about any possible implications in the build system changes to the respective platforms (like if all changes are correct), but this change in general looks good to me. My only remark would be that we now have `splash.jpg` and `splashscreen.png` which is kinda confusing. I suggest that we rename one of them to something else. Maybe `applaunch-screen.png`?

File: xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/Contents.json (hunk 1 above)
Comment (enen92) [reply xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/Contents.json:14]: There's a symlink on the folder

File: tools/webOS/packaging/appinfo.json.in (hunk 2 above)
Comment (enen92) [reply tools/webOS/packaging/appinfo.json.in:10]: Maybe...I added a symlink but I am unsure if it works.

Comment (sundermann) [reply tools/webOS/packaging/appinfo.json.in:10]: Doesn't seem to work: `ares-package ERR! [Tips]: The specified path does not exist <splashscreen.png>`

**Review by kambala-decapitator — APPROVED** [review]
iOS/tvOS changes look fine (haven't runtime tested them)

**Review by sundermann — APPROVED** [review]
The webOS part works fine (runtime tested)

**Comment by enen92** [comment]
Jenkins build this please

**Comment by enen92** [comment]
Runtime tested this on android last night, works well (both splashes are shown)

**Pull request merged**